docs/guide/plugins/plugins.md

```diff
@@ -78,32 +78,40 @@ end
 
 ```
 
+For more extensive examples of plugins, you can check out the code of the plugins bundled with Xournal++ [in the repository](https://github.com/xournalpp/xournalpp/tree/master/plugins).
+
 ## Plugin API
 
 The Lua Plugin can execute a number of Xournal++ functions to interact with Xournal++. Those are defined in the
-[Plugin API](https://github.com/xournalpp/xournalpp/blob/a92fcac0273a39fe89beeeb88c14406864c10306/src/plugin/luapi_application.h#L742-L762).
+[Plugin API](https://github.com/xournalpp/xournalpp/blob/9d1277dee22bb095c2db047bb04f89cc837aee3c/src/plugin/luapi_application.h#L877-L898).
 
 Currently the list contains the following functions:
 
-- `app.msgbox` for displaying messages to the user and letting the user choose an option by clicking a button
-- `app.saveAs` for retrieving the filename from a native saveAs dialog.
-- `app.registerUi` used in the `initUi`-function
-- `app.uiAction` for simulating a toolbar/menubar click starting some action,
-- `app.uiActionSelected` for notifying action listeners about selected options,
-- `app.changeCurrentPageBackground` for changing the background of the current page,
-- `app.sidebarAction` for actions accessible in the sidebar
-- `app.layerAction` for actions accessible in the layer controller
-- `app.changeToolColor` for changing the color of any tool with color capabilities,
-- `app.changePdfBackgroundPageNr` to change the pdf background page number
-- `app.getDocumentStructure` to get lots of useful info on the document, also used for applying operations on all/selected pages
-- `app.scrollToPage`scrolls relatively or absolutely to a page
+- `app.msgbox` shows a message box to the user and lets them choose an option by clicking a button
```

Comment:
The grammar should be fixed somehow. Not sure what is the best way.
```suggestion
- `app.msgbox` shows a message box to the user and lets the user choose an option by clicking a button
```

Reply:
How about this:
```suggestion
- `app.msgbox` displays a message box with the specified buttons and returns the button clicked by the user
```

or simply leave out the first mention of the user (who else would the msgbox be shown to?), I think "displays" fits better then:
```suggestion
- `app.msgbox` displays a message box and lets the user choose an option by clicking a button
```

Reply:
I like the first suggestion. It looks more precise than the latter one.